pkg/scheduler/plugins/reservation/plugin.go

```diff
@@ -492,6 +486,18 @@ func (pl *Plugin) filterWithPreAllocatablePods(ctx context.Context, cycleState *
 	preemptible := framework.NewResource(state.preemptible[node.Name])
 	state.preemptLock.RUnlock()
 
+	var insufficientResourcesByNodeUnallocated []string
```

Comment:
In the current implementation, this code will execute regardless. However, if there are Pods that can fulfill the PreAllocation requirement, this code will not need to execute.

Reply:
@ZiMengSheng Please let me know if there is any more concerns. Thanks!